packages/features/portfolio/src/containers/portfolio-txn-history/hooks/use-txn-history-data.tsx

```diff
@@ -14,7 +14,6 @@ export const useTxnHistoryData = (txnHistoryCategory: number) => {
   const theme = useTheme();
   let assetOrVaultData: SelectedOptions[] = [];
   let currencyData: SelectedOptions[] = [];
-
   const network = useSelectedNetwork();
```

Comment:
This hook should re-use the `use-formatted-data` from analytics view, I think we should also re-use the column configuration as well so the things are more or less the same.

In the per account view, however, the only change would be to remove the account column.